src/api/dto.ts

```diff
@@ -217,3 +217,15 @@ export type Role =
   | 'ROLE_INFRA_전산관리자'
   | 'ROLE_회원'
   | 'ROLE_출제자';
+
+export interface Job {
+  jobId: number;
+  jobName: Role;
+  jobThumbnailPath: string;
+}
+
+export interface Member extends Job {
+  memberId: number;
+  generation: number;
+  realName: string;
+}
```

Comment:
<img width="408" alt="image" src="https://github.com/KEEPER31337/Homepage-Front-R2/assets/78250089/53f22833-fed8-43c4-a49a-dd999e93dd01">

[여기 533PR](https://github.com/KEEPER31337/Homepage-Front-R2/pull/533/files#diff-fbab94b44da31fdcf3260650442bf8e8320bf72cc11a90b027862e052f9591cc)에서 멤버 정보에 대한 dto 작성해서 PR 올라가 있는 상태인데
PR 머지되면
```suggestion
export type ExecutiveMember = Pick<MemberDetailInfo, 'id' | 'generation' | 'realName'> & Job;
```
으로 타입 뽑아 사용하는 건 어떨까요?

Reply:
++ dto에 `MemberInfo`에 `id`로 되어 있는 부분 `id` -> `memberId`로 API 변경 요청해둔 상태입니다!

Reply:
허걱 넹넹!!! member에 속성이 굉장히 많군요! 나중에 수정하면 봐주쎄용!